tests/unit/s2n_pem_rsa_dhe_test.c

```diff
@@ -67,6 +67,7 @@ int main(int argc, char **argv)
     char *private_key_pem;
```

Comment:
While you're in here, consider
1. Zero init all the params
2. Move them to where they are actually defined.

Reply:
I zero initialized all  parameters.